src/main/java/frc/robot/subsystems/ShooterSubsystem.java

```diff
@@ -0,0 +1,106 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018-2019 FIRST. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.subsystems;
+
+import com.revrobotics.CANEncoder;
+import com.revrobotics.CANPIDController;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.ControlType;
+import com.revrobotics.CANSparkMax.IdleMode;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+import java.util.logging.*;
+
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+/**
+ * Add your docs here.
+ */
+public class ShooterSubsystem extends SubsystemBase{
+    private CANSparkMax mShoot;
+    private CANPIDController pidController;
+    private CANEncoder encoder;
+    public double kP, kI, kD, kIz, kFF, kMaxOutput, kMinOutput;
+    private static final Logger LOGGER = Logger.getLogger(DriveSubsystem.class.getName());
+
+
+    public ShooterSubsystem() {
+        mShoot = new CANSparkMax(1, MotorType.kBrushless);
+        mShoot.restoreFactoryDefaults();
+        mShoot.enableVoltageCompensation(12);
+        mShoot.setIdleMode(IdleMode.kBrake);
+
+        pidController = mShoot.getPIDController();
+        encoder = mShoot.getEncoder();
+
+        kP = 0.00010; 
+        kI = 0;
+        kD = .0000; 
+        kIz = 0; 
+        kFF = 0.000175; 
+        kMaxOutput = 1; 
+        kMinOutput = -1;
```

Comment:
Create shooter constants in the Constants.java file like Andrew said. Assign the constants as the defaults here.